### ISSUE TITLE
Expand/collapse state of the buttons is missing or incorrect

### ISSUE THREAD

**Issue by mshubat** [ableplayer/ableplayer#527]
The expand/collapse state of a toggle-type element is missing for
-CC button
-Preferences button
-Volume button

Note: The expanded and collapsed state is not conveyed to the screenreader users when expanded and collapsed 
- CC button announces expanded when it is in the collapsed state
- Preference button does not announce the expand and collapse state
- Volume button only announces collapsed state 

See: https://www.w3.org/TR/wai-aria-1.1/#aria-expanded

![captions](https://user-images.githubusercontent.com/10766136/173676185-7a01a436-8f49-4a1e-8a41-d35967ec52d1.png)
![preferences](https://user-images.githubusercontent.com/10766136/173676190-ee2cc1f6-d186-4dbb-a630-9aaf0e659d19.png)
![volume](https://user-images.githubusercontent.com/10766136/173676192-81c80eab-5b99-43b7-92d9-ce367d22b872.png)

**Comment by terrill** [comment]
Closed with 6e3899d0f8841d0e4f19738dff2219dc8dc07935
